security/selinux-tests.py

```diff
@@ -0,0 +1,95 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2020 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import archive, distro, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class SELinux(Test):
+
+    """
+    selinux-testsuite
+    :avocado: tags=security,testsuite
+    """
+
+    def setUp(self):
+        '''
+        Install the basic packages to support selinux
+        '''
+
+        # Check for basic utilities
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        deps = ['gcc', 'make']
+        if detected_distro.name in ['rhel', 'SuSE', 'fedora', 'centos',
+                                    'redhat']:
+            deps.extend(["perl-Test", "perl-Test-Harness", "perl-Test-Simple",
+                         "perl-libs", "selinux-policy-devel",
+                         "net-tools", "netlabel_tools", "iptables", "libbpf",
+                         "lksctp-tools-devel", "attr", "libbpf-devel",
+                         "keyutils-libs-devel", "quota", "xfsprogs-devel",
+                         "libuuid-devel", "nftables", "kernel-devel",
+                         "kernel-modules", "perl-Test-Harness", "coreutils",
+                         "netlabel_tools", "libsepol", "checkpolicy",
+                         "libselinux", "policycoreutils", "libsemanage",
+                         "nfs-utils", "policycoreutils-newrole",
+                         "xfsprogs-devel", "libselinux-devel"])
+        else:
+            self.cancel("Install the package for perf supported\
+                      by %s" % detected_distro.name)
+        for package in deps:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.cancel('%s is needed for the test to be run' % package)
+
+        url = "https://github.com/SELinuxProject/selinux-testsuite/archive/master.zip"
+
+        tarball = self.fetch_asset(url, expire='7d')
+        archive.extract(tarball, self.workdir)
+        self.sourcedir = os.path.join(self.workdir, 'selinux-testsuite-master')
+        os.chdir(self.sourcedir)
+        output = process.system_output('getenforce', ignore_status=True)
+        if output.decode() == 'Permissive':
+            process.run('setenforce 1', ignore_status=True)
+            output = process.system_output('getenforce', ignore_status=True)
+            if output.decode() == 'Permissive':
+                self.fail("Unable to enter in to 'Enforcing' mode")
+        status, output = process.getstatusoutput('make -C policy load',
```

Comment:
build.run_make can be used and we can use `<variable>.exit_status` and `<variable>.stdout` for next processing 

for more info refer below 

https://github.com/avocado-framework/avocado/blob/master/avocado/utils/build.py#L51